frontend/app/component/form/differences/DifferencesForm.test.tsx

```diff
@@ -202,40 +199,41 @@ describe("Test DifferencesForm", () => {
       expect(url).toEqual("http://testhost/api/polling_stations/1/data_entries/1");
       expect(method).toEqual("POST");
       expect(body).toEqual(expectedRequest);
-
-      const result = await screen.findByTestId("result");
-      expect(result).toHaveTextContent(/^Success$/);
-    });
-
-    test("422 response results in display of error message", async () => {
-      overrideOnce("post", "/api/polling_stations/1/data_entries/1", 422, {
-        message: "422 error from mock",
-      });
-
-      const user = userEvent.setup();
-
-      renderForm();
-
-      const submitButton = await screen.findByRole("button", { name: "Volgende" });
-      await user.click(submitButton);
-      const feedbackServerError = await screen.findByTestId("feedback-server-error");
-      expect(feedbackServerError).toHaveTextContent(/^Error422 error from mock$/);
     });
 
-    test("500 response results in display of error message", async () => {
-      overrideOnce("post", "/api/polling_stations/1/data_entries/1", 500, {
-        message: "500 error from mock",
-      });
-
-      const user = userEvent.setup();
-
-      renderForm();
-
-      const submitButton = await screen.findByRole("button", { name: "Volgende" });
-      await user.click(submitButton);
-      const feedbackServerError = await screen.findByTestId("feedback-server-error");
-      expect(feedbackServerError).toHaveTextContent(/^Error500 error from mock$/);
-    });
+    //Server errors are no longer rendered inside the form
+    // test("422 response results in display of error message", async () => {
+    //   overrideOnce("post", "/api/polling_stations/1/data_entries/1", 422, {
+    //     message: "422 error from mock",
+    //   });
+    //
+    //   const user = userEvent.setup();
+    //
+    //   renderForm();
+    //
+    //   const submitButton = await screen.findByRole("button", { name: "Volgende" });
+    //   await user.click(submitButton);
+    //   //TODO: server errors moved out of the form
+    //   //const feedbackServerError = await screen.findByTestId("feedback-server-error");
+    //   //expect(feedbackServerError).toHaveTextContent(/^Error422 error from mock$/);
+    // });
+
+    //Server errors are no longer rendered inside the form
+    // test("500 response results in display of error message", async () => {
+    //   overrideOnce("post", "/api/polling_stations/1/data_entries/1", 500, {
+    //     message: "500 error from mock",
+    //   });
+    //
+    //   const user = userEvent.setup();
+    //
+    //   renderForm();
+    //
+    //   const submitButton = await screen.findByRole("button", { name: "Volgende" });
+    //   await user.click(submitButton);
+    //   //TODO: server errors moved out of the form
+    //   //const feedbackServerError = await screen.findByTestId("feedback-server-error");
+    //   //expect(feedbackServerError).toHaveTextContent(/^Error500 error from mock$/);
+    // });
```

Comment:
it's rendered on the same page but not by the form.
PollingStationFormNavigation renders it

Reply:
I see! I'll clean up the tests.

Reply:
Those commented out tests have an `OverrideOnce()` with a response that does not match the type `ErrorResponse`. That invalid response does match with what `PollingStationFormNavigation.tsx` expects: `{error.errorCode}: {error.message}`. (discovered this because I was looking into something else)

Reply:
Fixed in 6b1435ec0ceb41fcb42609fef9d8b2fc2158a9e5

Reply:
We need a better test for server errors, because if the backend returns a 422 with body `{"error": "some error text here"}, you get this in the frontend:

![image](https://github.com/user-attachments/assets/14e1f4c9-e0d9-4c86-95ef-ea7dee8bb14d)

(Also note that the styling seems off: error box looks a bit close to the heading.)

Reply:
The server error is improved in https://github.com/kiesraad/abacus/pull/266 please check if you have any feedback on what exact text it should show there

Reply:
> We need a better test for server errors, because if the backend returns a 422 with body `{"error": "some error text here"}, you get this in the frontend:
> 
> [..]

This should also be fixed in 6b1435ec0ceb41fcb42609fef9d8b2fc2158a9e5 because it parses the API POST response correctly.